### PR TITLE
Add Ruby 3.2 to CI

### PR DESCRIPTION
**Pull request by digitalmoksha**

### CODE REVIEW

**Comment by digitalmoksha** [comment]
@jirutka I think this ready for review/merging

File: asciidoctor-include-ext.gemspec [hunk 1]
```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop', '~> 0.51.0'
   s.add_development_dependency 'simplecov', '~> 0.15'
   s.add_development_dependency 'yard', '~> 0.9'
+  s.add_development_dependency 'webrick', '~> 1.8'
```
Comment (jirutka) [inline asciidoctor-include-ext.gemspec:32]: Please split this into a separate commit.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -27,6 +28,7 @@ jobs:
           - { asciidoctor: 1.5.7     , ruby: 2.6 }
           - { asciidoctor: 1.5.6.2   , ruby: 2.6 }
           - { asciidoctor: 'git:main', ruby: 3.1 }
+          - { asciidoctor: 'git:main', ruby: 3.2 }
```
Comment (jirutka) [inline .github/workflows/ci.yml:30]: Remove this and just bump the `ruby` version in the line above.

**Comment by digitalmoksha** [comment]
No problem, done!

**Comment by jirutka** [comment]
Thanks!

**Comment by digitalmoksha** [comment]
Thanks for the quick turn-around!
